app/src/main/java/com/codepath/travel/models/StoryPlace.java

```diff
@@ -43,6 +48,15 @@ public StoryPlace(Trip trip, Place place) {
         setLatitude(latlng.latitude);
         setLongitude(latlng.longitude);
         setPlaceTypes(place.getPlaceTypes());
+        setRating(place.getRating());
```

Comment:
Sure! I have used it for now since I had to show the place rating. Or do we not want to show google rating? We can have another field for showing user rating when we implement the functionality.